<launch>

<node pkg="prophesee_ros_driver" type="stereo_publisher" name="stereo_publisher" respawn="false" output="screen">
  <param name="camera_name" value="camera" />
  <param name="left_camera_id" value="00002400" />
  <param name="right_camera_id" value="00002402" />
  <param name="master_left" value="true" />
  <param name="publish_cd" value="true" />
  <param name="publish_graylevels" value="false" />
  <param name="publish_imu" value="false" />

  <!-- Path to the file with the camera settings -->
  <param name="bias_file" value="" />

  <!-- Maximum event rate in kEv/s -->
  <param name="max_event_rate" value="6000" />

  <!-- Graylevel frame rate in fps -->
  <param name="graylevel_frame_rate" value="30" />
</node>

</launch>
